renderer/utils/datetime-ui.ts

```diff
@@ -177,3 +177,16 @@ export const convertMillisecondsToTime = (milliseconds) => {
 
   return `${hours}:${minutes}`;
 };
+
+export const getCurrentTimeRoundedUp = () => {
+  const currentTime = new Date();
+  const minutes = currentTime.getMinutes();
+  const minutesToAdd = (15 - (minutes % 15)) % 15;
+  currentTime.setMinutes(minutes + minutesToAdd);
+  const formattedTime = currentTime.toLocaleTimeString([], {
+    hour: "2-digit",
+    minute: "2-digit",
+  });
+
+  return `${formattedTime}`;
+};
```

Comment:
I have "AM/PM" when ctrl+d pressed. You should get rid of them 
![image](https://github.com/ukad-group/timetracker-desktop-client/assets/110410115/80f69fc8-d8d2-4bd1-8114-2394e60bdfd0)